### DISCUSSION TITLE
Terraform `1.8.2` ships additional file within tarball

### DISCUSSION THREAD

**Discussion by yermulnik** [warrensbox/terraform-switcher#411] [Ideas]
Should we be taking care of this (e.g. to not vomit `.terraform.versions` dir with arbitrary file)?

https://github.com/hashicorp/terraform/releases/tag/v1.8.2
NOTE:

Starting with this release, we are including a copy of our license file in all packaged versions of our releases, such as the release .zip files. If you are consuming these files directly and would prefer to extract the one terraform file instead of extracting everything, you need to add an extra argument specifying the file to extract, like this:

```shell
unzip terraform_1.8.2_linux_amd64.zip terraform
```

**Comment by yermulnik** [comment]
@warrensbox @MatrixCrawler @crablab @MatthewJohn @jukie

**Reply by yermulnik** [reply]
🤣 
![image](https://github.com/warrensbox/terraform-switcher/assets/1274789/90f20af0-edb4-4778-95cb-72167210b200)

**Comment by MatrixCrawler** [comment]
I agree that we don't need to clutter the .terraform.versions directory.
Let's create an issue for that.

**Comment by warrensbox** [comment]
Thanks for bring this up. Yeah, I agree as well!

**Comment by MatthewJohn** [comment]
See https://github.com/warrensbox/terraform-switcher/issues/414 and https://github.com/warrensbox/terraform-switcher/pull/433
